dyno-core/src/main/java/com/netflix/dyno/connectionpool/HostSupplier.java

```diff
@@ -30,5 +30,5 @@
 	 * Return a collection of dynomite hosts for the connection pool
 	 * @return Collection<Host>
```

Comment:
Please update the Javadoc to contain `List` since this the structure of your preference...  Note that a `Collection` is more generic.

Reply:
ok. The thing is we are everywhere taking a collection and converting it to a string etc.

Reply:
done